docker-compose.yml

```diff
@@ -60,7 +60,7 @@ services:
       timeout: 5s
       retries: 5
     ports:
-      - "5432:5432"
+      - "${FORMANCE_POSTGRES_PORT:-5432}:5432"
```

Comment:
If `FOMANCE_HOST_PORT` is not set, `docker compose` will fall back to forwarding to port `5432`, resulting in the same behavior as without this change for anyone not using this configuration option. The `-` tells `docker compose` to also use the default, if the environment variable is available but left empty.

I'm open to suggestions for the naming. If there is any existing convention I'd be happy if you could point it out to me.